### ISSUE TITLE
Read me update to explain how to actually produce the components necessary to have a new fully operation section

### ISSUE THREAD

**Issue by akramsarkis** [Geeks-Solutions/nuxt-sections#55]
- [ ] Explain how to actually produce the components necessary to have a new fully operation section (whether its local, static, dynamic or configurable)

- It lacks the required variable in the data of the view component (for static, dynamic and configurable)
- It lacks the required format to get data sent to sections for static section types forms
- it lacks the mechanism we defined to customize the display of form elements for configurable section types forms

Basically we need to cover these topics with links to examples (from the various sections we already created, so it's clear for any dev on how to produce new section components

### **Acceptance testing**

- [ ] Have Julien validate the read me

**Comment by akramsarkis** [comment]
https://gl.eweev.com/eweev/geeks/sections/-/wikis/home
